### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by zorkow**
The (provisional) end of the AT project with MathJax.

### CODE REVIEW

File: src/common/mathjax.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ goog.require('sre.System');
 
   (sre.System.getInstance()).setupEngine(
       {mode: sre.Engine.Mode.HTTP,
-       json: MathJax.Ajax.config.path['SRE'],
+       json: MathJax.Ajax.config.path['SRE'] + '/mathmaps/',
```
Comment (pkra) [inline src/common/mathjax.js]: I think this should be 

`json: MathJax.Ajax.config.path['SRE'] + 'mathmaps/',`

Without the leading `/` for `mathmaps`.

Comment (pkra) [inline src/common/mathjax.js]: cc @dpvc

Comment (pkra) [inline src/common/mathjax.js]: Nonsense. The other way around:

`json: MathJax.Ajax.config.path['SRE'] + '/mathmaps',`

Right?

Comment (dpvc) [inline src/common/mathjax.js]: No, the way SRE is using it, it needs both slashes, as Volker has originally.

Comment (pkra) [inline src/common/mathjax.js]: Ok. Still, I'm getting errors from bad URLs, e.g., https://beta.mathjax.org/mathjax/contrib/a11y/mathmaps//symbols/latin-upper-normal.json

Comment (pkra) [inline src/common/mathjax.js]: And making the second proposed change locally fixes that.

Comment (zorkow) [inline src/common/mathjax.js]: Fixed now. PTAL.

Comment (dpvc) [inline src/common/mathjax.js]: It looks like `jsonPath` is used in two places, but inconsistently.

* [browser_tule.js line 105](https://github.com/zorkow/speech-rule-engine/blob/259d350ad02f47b8373c62c76f23f1e47aefc342/src/common/browser_util.js#L105) needs the extra `/` at the end, but

* [math_map.js line 172](https://github.com/zorkow/speech-rule-engine/blob/259d350ad02f47b8373c62c76f23f1e47aefc342/src/mathmaps/math_map.js#L172) adds it itself (giving you the double `//` in your example).

Comment (zorkow) [inline src/common/mathjax.js]: Ah, thanks for finding that. I always wanted to have a utility function to take care of concatenating path names. Maybe it's time now.

**Pull request merged**